### DISCUSSION TITLE
Tag listings on hub.docker.com

### DISCUSSION THREAD

**Discussion by sbmw** [dotnet/dotnet-docker#3173] [Q&A] [answered]
Hi, is it intentional that the hub.docker.com pages now only list high level tags such as 5.0 rather than eg 5.0.401-alpine3.14 in the case of https://hub.docker.com/_/microsoft-dotnet-sdk?  There is no mention of the various distro-based tags at all.

A link to https://mcr.microsoft.com/v2/dotnet/sdk/tags/list is given for a "Full Tag Listing" but the sort order is unclear.

**Comment by richlander** [comment, marked as answer]
Thanks for reporting. That looks like a bug and is unintentional.

@mthalman

**Reply by mthalman** [reply]
I've reported this to the MCR team since it seems to be an issue in how they are publishing to Docker Hub.

**Comment by sbmw** [comment]
Awesome, many thanks.
